🗃 Start admonitions in the open position (#1486)
jupyter-book/mystmd: docs/admonitions.md

To turn an admonition into a dropdown, add the `dropdown` {myst:directive}`admonition.class` to them.
Dropdown admonitions use the `<details>` HTML element (meaning they also will work without Javascript!),
and they can be helpful when including text that shouldn't immediately visible to your readers.

> To have the admonition be a dropdown, and also start open, add the {myst:directive}`admonition.open` option.

Reviewer: ```suggestion
To have a dropdown-style admonition start open, add the {myst:directive}`admonition.open` option.
```

Reviewer: The `open` option doesn't add the `class` for dropdown, right?

Reviewer: Perhaps we should add the `dropdown` class if `open` is defined?

Reviewer: @rowanc1 should we warn the user if they don't define `class: dropdown` then?

Author: Yeah - I will add that. Right now it just doesn't do anything silently.
